### ISSUE TITLE
[Android 0.20 - ReactART] Groups are being "painted" in wrong positions

### ISSUE THREAD

**Issue by lucasterra** [facebook/react-native#5466]
As the title says, ReactART on Android is painting groups in wrong positions.

The code below has two Surfaces (black bordered and red bordered).
On the first surface I draw 3 texts directly, without groups. They're all painted as expected.
On the second surface I draw them inside groups. They're painted at wrong positions.

I have included a screenshot to show you what I mean.

```javascript
'use strict';

import React, {
  AppRegistry,
  View,
  Dimensions
} from 'react-native';

import ReactART, {
  Surface,
  Group,
  Text
} from 'ReactNativeART';

var MyApp = React.createClass({
  render: function() {
    const dim = Dimensions.get('window');
    const font = `13px "Helvetica Neue", "Helvetica", Arial`;

    return (
      <View>
        {/* Draw three texts without groups */}
        <View style={{borderColor: 'black', borderWidth: 2}}>
          <Surface width={dim.width} height={200}>
            <Text font={font} fill="#000000" x={16} y={16}>Hello</Text>
            <Text font={font} fill="#000000" x={16} y={32}>Hello</Text>
            <Text font={font} fill="#000000" x={16} y={48}>Hello</Text>
          </Surface>
        </View>

        {/* Draw same thing with groups */}
        <View style={{borderColor: 'red', borderWidth: 2}}>
          <Surface width={dim.width} height={200}>
            <Group x={16} y={16}>
              <Text font={font} fill="#000000">Hello</Text>
            </Group>
            <Group x={16} y={32}>
              <Text font={font} fill="#000000">Hello</Text>
            </Group>
            <Group x={16} y={48}>
              <Text font={font} fill="#000000">Hello</Text>
            </Group>
          </Surface>
        </View>
      </View>
    );
  }
});

AppRegistry.registerComponent('MyApp', () => MyApp);
```

![android canvas](http://i.imgur.com/w3E1G6f.png "Android Canvas")

**Comment by lucasterra** [comment]
I've also made a post on [Product Pains](https://productpains.com/post/react-native/reactart-paints-groups-in-wrong-position-on-android/)

**Comment by lucasterra** [comment]
This still happens in 0.20

**Comment by lucasterra** [comment]
This was fixed in 0.22.x
